announcements/2021/week20.md

```diff
@@ -0,0 +1,30 @@
+# :zap: This week in provider-independent announcements (May 21, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
```

Comment:
sidenote: we recently realized that what we write in these announcements is much nicer than what we have in the changes and releases, cause those are just from the changelogs, nothing to do on this right now, but we should see if at some point we can better integrate the two

Reply:
yes! that's what this issue https://github.com/giantswarm/giantswarm/issues/17071 is about right?